.travis.yml

```diff
@@ -1,105 +1,42 @@
-language: php
+version: ~> 1.0
 
-dist: xenial
-
-services:
-  - mysql
-  - postgresql
-  - xvfb
-
-cache:
-  directories:
-    - $HOME/.composer/cache/files
-
-addons:
-  apt:
-    packages:
-      - tidy
-      - chromium-chromedriver
-      - chromium-browser
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/standard-jobs-fixed.yml
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=1.x-dev
-    - SS_BASE_URL="http://localhost:8080/"
-    - SS_ENVIRONMENT_TYPE="dev"
-    - RECIPE_VERSION=4.x-dev
+    - COMPOSER_ROOT_VERSION="1.7.x-dev"
+    - REQUIRE_RECIPE="4.7.x-dev"
 
-matrix:
-  fast_finish: true
+jobs:
```

Comment:
Could you add a small explanation about the need for a manual job matrix? I'm assuming this has to do with the use of `silverstripe/installer`, but would be useful for developers looking to adjust in the future.

Reply:
Have added explanation